docs/index.asciidoc

```diff
@@ -59,6 +60,14 @@ output plugins.
 
 Your DatadogHQ API key. https://app.datadoghq.com/account/settings#api
 
+[id="plugins-{type}s-{plugin}-api_url"]
+===== `api_url` 
+
+  * Value type is <<string,string>>
+  * Default value is `"https://api.datadoghq.com/api/v1/series"`
+
+Set the api endpoint for Datadog EU Site users ("https://api.datadoghq.eu/api/v1/series")
```

Comment:
Is there a way we can link to their list of endpoints instead of owning a list here?

Reply:
I spent some time investigating this and reached out to the Docs team at Datadog and it doesn't look like this is documented canonically anywhere at the moment. Given that i've removed the explicit reference to the endpoint as it could drift in the future. I think Datadog plans on adding a list of canonical endpoints to their docs in the future, so can keep an eye on that and add to the docs when that is released

Reply:
@yaauie not quite sure why the CLA integration check is stuck in "pending" for me as I've signed it